docs/tedx.md

```diff
@@ -23,16 +23,18 @@ Use letra maiúscula apenas na primeira palavra do título, em nomes próprios e
 
 > “A história da eletricidade | Palestrante Fictício | TEDxQualquer” em vez de “A História Da Eletricidade ...”
 
-
-
-
 ### Descrição
-A descrição das palestras TEDx deve ser um breve resumo da palestra. Caso haja links para outros sites, exclua-os (exceto se forem da organização do palestrante, se esta for objeto da palestra). Se a descrição contiver a biografia do palestrante, você pode mantê-la, **mas exclua o texto que fala sobre o que é o programa TEDx** (“In the spirit of ideas worth spreading, TEDx is a program of local, self-organized events…”).
+A descrição das palestras TEDx deve ser um breve resumo destas. Caso haja links para outros sites, exclua-os (exceto se forem da organização do palestrante, se esta for objeto da palestra). Se a descrição contiver a biografia do palestrante, você pode mantê-la, **mas exclua o texto que fala sobre o que é o programa TEDx** (“In the spirit of ideas worth spreading, TEDx is a program of local, self-organized events…”).
+
+Disclaimer das palestras TEDX:
```

Comment:
Não acho que inserir o disclaimer aqui seja bom. Já há uma referência a onde encontrá-lo na linha 36, com um link para o glossário. Colocar mais uma versão aqui só faz com que tenhamos mais trabalho de procurar onde arrumar toda vez que ele for atualizado.